nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -148,24 +147,23 @@ def _apply(self,
                 output_fp=output_fp,
                 output_name=output_name)
 
-            current_bias = self._backend_entity.get_bias_value(model, node)
-            updated_bias = current_bias + bias_shift
-            magnitude = self._get_bias_shift_magnitude(current_bias, updated_bias)
+            updated_bias = bias_value + bias_shift
+            magnitude = self._get_bias_shift_magnitude(bias_value, updated_bias)
 
             if magnitude < self.threshold:
                 nncf_logger.debug(f'{node_name} bias would be changed')
-                bias_port_id = self._backend_entity.get_bias_port_id(model, node)
-                target_point = self._backend_entity.target_point(TargetType.LAYER,
-                                                                 node.node_name,
-                                                                 bias_port_id)
-                bias_correction_command = self._backend_entity.bias_correction_command(target_point,
-                                                                                       updated_bias)
-                transformation_layout.register(bias_correction_command)
+                node_and_new_bias_value.append((node, updated_bias))
             else:
                 nncf_logger.debug(f'{node_name} bias skipped by threshold')
 
-        quantized_model = model_transformer.transform(transformation_layout)
-        return quantized_model
+        # Create commands of bias correction and apply them to the model.
```

Comment:
What is the difference/profit between the previous code?

Reply:
The main purpose of this change is to separate the command creation logic and the logic of the new bias calculation. It makes the code a little bit easy.